HousingManagementSystemApi.Tests/HelpersTests/CosmosAddressQueryHelperTests.cs

```diff
@@ -0,0 +1,64 @@
+namespace HousingManagementSystemApi.Tests.HelperTests
+{
+    using HACT.Dtos;
+    using Helpers;
+    using Microsoft.Azure.Cosmos;
+    using Moq;
+    using Xunit;
+
+    public class CosmosAddressQueryHelperTests
+    {
+        private CosmosAddressQueryHelper systemUnderTest;
+        private Mock<Container> containerMock;
+        private Mock<FeedIterator<PropertyAddress>> feedIteratorMock;
+
+        public CosmosAddressQueryHelperTests()
+        {
+            this.containerMock = new Mock<Container>();
+            this.systemUnderTest = new CosmosAddressQueryHelper(this.containerMock.Object);
+            this.feedIteratorMock = new Mock<FeedIterator<PropertyAddress>>();
+        }
+
+        [Fact]
+        public void Test_Query_Definition_returned_with_querytext()
```

Comment:
Correct test name casing
```suggestion
        public void Test_Query_Definition_Returned_With_QueryText()
```